pkgs/development/compilers/graalvm/graalvm-oracle/hashes.nix

```diff
@@ -0,0 +1,42 @@
+{
+  "22" = {
+    "hashes" = {
```

Comment:
why not flatten `hashes` attr into the parent attrset? also, why not use SRI hashes (`hash` instead of `sha256`)?